run_calculate_loanbook_coverage.R

```diff
@@ -9,6 +9,7 @@ library(vroom)
 
 dotenv::load_dot_env()
 source("expected_columns.R")
+source("functions_prep_project.R")
```

Comment:
NIT: Consider moving files that are sourced exclusively for functions into an `R` folder. They are all likely candidates for extraction into a package.

Note: I am assuming that's what these files contain only functions and not actual scripts.